apiserver/application/application_test.go

```diff
@@ -1480,6 +1480,91 @@ func (s *applicationSuite) TestApplicationUpdateInvalidApplication(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `application "no-such-application" not found`)
 }
 
+func (s *applicationSuite) TestApplicationUpdateSeries(c *gc.C) {
```

Comment:
New tests should be added to application_unit_test.go
The tests in application_test.go use JujuConnSuite which we are trying to move away from